friends/src/Components/Home/Home.css

```diff
@@ -0,0 +1,39 @@
+body {
+  /* margin: 0 auto;
+  height: 100%;
+  width: 100%; */
+  background-image: url("./img/background.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  /* display: flex;
+  justify-content: center; */
+}
+
+h1 {
+	font-family: Orbitron;
+	font-size: 70px;
+	font-style: italic;
+	font-variant: small-caps;
+	font-weight: 700;
+}
+
+.FriendsDiv {
+    height: 100vh;
```

Comment:
add your background image inside this block to affect only the friends div section and not the whole HTML body.